src/components/Release/Release.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { useFetchSongs } from '../../services/hooks';
+import withList from '../HOCs/WithList/WithList';
+import Song from '../Songs/Song';
+import placeholder from '../../assets/placeholder.png';
+import { useParams } from 'react-router-dom';
+import style from './release.css';
+
+const Release = () => {
+  const { releaseTitle, releaseId, artist } = useParams();
+
+  const img = `http://coverartarchive.org/release/${releaseId}/front`;
+  const handleImgError = ({ target }) => (target.src = placeholder);
+  
+  const songsData = useFetchSongs(releaseId, artist);
+  const Songs = withList(Song);
```

Comment:
Nice use of Higher-Order Components. Starting to think I should've tried that out.